awx/api/serializers.py

```diff
@@ -3009,7 +3009,7 @@ class Meta:
         fields = ('*', 'host_config_key', 'ask_diff_mode_on_launch', 'ask_variables_on_launch', 'ask_limit_on_launch', 'ask_tags_on_launch',
                   'ask_skip_tags_on_launch', 'ask_job_type_on_launch', 'ask_verbosity_on_launch', 'ask_inventory_on_launch',
                   'ask_credential_on_launch', 'survey_enabled', 'become_enabled', 'diff_mode',
-                  'allow_simultaneous', 'custom_virtualenv')
+                  'allow_simultaneous', 'custom_virtualenv', 'job_shard_count')
```

Comment:
In the JT serializer, we should considering throwing a 400 if `allow_simultaneous` is false and `job_shard_count` is greater than 1.

I can't envision a reasonable use case for this configuration of linearizing a JT, and I'd predict some user confusion if we allow it.

Reply:
split, plz.

Reply:
+1

Reply:
I'm still gathering the totality of renames before implementing them, then going through the process of migrating and integration tests. This is the number of jobs to split into, so the field will probably wind up being `job_split_count`, because I think the UI has been going in that direction.

Also, I think the field pointing from the workflow job to the job template would be best named `split_job_template` as well. This will help keep the naming schemes consistent.